packages/android_alarm_manager/example/pubspec.yaml

```diff
@@ -4,8 +4,7 @@ description: Demonstrates how to use the android_alarm_manager plugin.
 dependencies:
   flutter:
     sdk: flutter
-  android_alarm_manager:
-    path: ../
+  android_alarm_manager: {path: ../} # ^x.y.z in your application
```

Comment:
I really don't care if this is one line or 2 lines, though I don't want these to go back-and-forth between the 2 styles.
This is the kind of things we should write in a style guide, the closest thing to a style guide I found are the examples here: https://dart.dev/tools/pub/dependencies (which are using 2 lines). If we believe this style is preferred we should probably update that page as well.

Reply:
I only used the one-liner style because it put the comment on the same line.